lib/config.js

```diff
@@ -85,13 +85,20 @@ function loadConfig(filePath) {
         } else {
 
             // it's a package
-            if (filePath.indexOf("eslint-config-") === -1) {
-                if (filePath.indexOf("@") === 0) {
+
+            if (filePath.charAt(0) === "@") {
+                // it's a scoped package
+
+                // package name is "eslint-config", or just a username
+                var scopedPackageShortcutRegex = /^(@[^\/]+)(?:\/(?:eslint-config)?)?$/;
+                if (scopedPackageShortcutRegex.test(filePath)) {
+                    filePath = filePath.replace(scopedPackageShortcutRegex, "$1/eslint-config");
+                } else if (filePath.indexOf("eslint-config-") === -1) {
                     // for scoped packages, insert the eslint-config after the first /
-                    filePath = filePath.replace(/^([^\/]+\/)(.*)$/, "$1eslint-config-$2");
-                } else {
-                    filePath = "eslint-config-" + filePath;
+                    filePath = filePath.replace(/^@([^\/]+)\/(.*)$/, "@$1/eslint-config-$2");
                 }
+            } else if (filePath.indexOf("eslint-config-") === -1) {
```

Comment:
This is the same as the existing logic - I don't want to change non-scoped-package names in this PR.